tests/Core/Ruleset/ProcessRulesetShouldProcessElementTest.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="ProcessRulesetShouldProcessElementTest" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/PHPCSStandards/PHP_CodeSniffer/master/phpcs.xsd">
+
+    <file>.</file>
+
+	<!--
```

Comment:
One tiny detail is the indentation of these comment blocks. Initially, I thought that you were intentionally using two levels of indentation for them for some reason, as more than one PR uses this pattern. But I just noticed that in #715, the comment blocks are using one level of indentation, so I'm not sure anymore:

https://github.com/PHPCSStandards/PHP_CodeSniffer/pull/715/files#diff-35fd5542d976e814f5d13530e285cbe46758aa143caced48522b53c4fae2e81eR6-R11

Reply:
🤦🏻 Sorry about that, some tabs had sneaked in. Fixed now.

Reply:
Thanks!